### PR TITLE
Mimes can rename themselves

### PR DESCRIPTION
**Pull request by uraniummeltdown**
Mimes can now choose their own name on entry just like Clowns

:cl:
add: Mimes can now choose name on entry just like Clowns
/:cl:

### CODE REVIEW

**Comment by Aurorablade** [comment]
I dunno I mean I interesting concept but I never saw why mimes needed to swap names

**Comment by Dumbdumn5** [comment]
I mean, I don't see why not, the clowns get their silly names, why not the mimes too?

**Comment by Spacemanspark** [comment]
Mimes have rights

File: config/names/mime.txt [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Invisible Man
+Lemon Mime
```
Comment (Crazylemon64) [inline config/names/mime.txt]: :eyes:

**Pull request merged**